### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by ericms35** [OlivierZal/com.melcloud#554] [Announcements]
Hi Olivier.  Thank you for your job, it’s great. I just installed a new Ecocan pac and I m not not pretty sure it works fine. So Jomey and your app are a great help for me to try to analyse how my HP works. Can you just tell me where I can find all details about parameters in your app? Cause I don’t understand all of them. For instance, what is frequency? What différence between cop and daily cop?  Than you for kind help. Eric

**Comment by OlivierZal** [comment]
Hi @ericms35, thanks for your nice feedback!

You can find a **technical** documentation of the MELCloud API [here](https://github.com/OlivierZal/melcloud-api/pkgs/npm/melcloud-api), but unfortunately I confess I can't help more: I have myself an air-to-air heat pump, and got help from other users to make the air-to-water implementation work.

However you can probably find experts able to answer you [on the forum](https://community.homey.app/t/app-pro-melcloud-melcloud-extension-take-control-of-your-heat-pump/73094/282).
